test/SILGen/dynamically_replaceable.swift

```diff
@@ -211,34 +211,34 @@ extension Klass {
   }
 }
 
-extension Strukt {
+extension Struct {
```

Comment:
It's possible this is a reserved word...

Reply:
`Struct` with an uppercase `S` appears to not be reserved. However, given the class decl `Klass` above I think the spelling here seems intentional enough to me that I would drop this from the PR.

Reply:
Hmm, looks like I failed to drop this. I can...